### PR TITLE
Suspend the whole tokenizer and tree builder on </script>

### PR DESCRIPTION
**Pull request by nox**
This change is [Reviewable](https://reviewable.io/reviews/servo/html5ever/231)

### CODE REVIEW

File: src/tokenizer/mod.rs [hunk 1]
```diff
@@ -404,19 +409,19 @@ impl<Sink: TokenSink> Tokenizer<Sink> {
             self_closing: self.current_tag_self_closing,
             attrs: replace(&mut self.current_tag_attrs, vec!()),
         });
-        self.process_token(token);
+        ;
```
Comment (Ms2ger) [inline src/tokenizer/mod.rs]: ?

Comment (nox) [inline src/tokenizer/mod.rs]: Potato fingers. I wonder how the tests passed though...

**Comment by SimonSapin** [comment]
@bors-sero r+

---

Reviewed 8 of 9 files at r1, 1 of 1 files at r2, 3 of 3 files at r3, 1 of 1 files at r4.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/servo/html5ever/231#-:-KVevqmUXunmjPNrD68v:b-asurui)*

**Comment by nox** [comment]
@bors-servo r-

I am dumb, you can't know which script halted the tokenizer as the code is currently.

**Comment by SimonSapin** [comment]
@bors-servo r+

---

Reviewed 10 of 10 files at r5.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/servo/html5ever/231#-:-KW2AJey-HoyPmJlGuJG:b-59p4e2)*

**Comment by Ygg01** [comment]
Does this closes #7 #19 ?

**Pull request merged**